### PR TITLE
Create liststeps-endpoint-deprecation.md

### PR DESCRIPTION
**Pull request by mirnawong1**
release note for list steps deprecation warning 

https://www.notion.so/dbtlabs/Deprecation-Warning-of-List-Steps-API-Endpoint-796ec4a94d784cfab2d4cf5fae0e6d94

## Prerelease docs
If this change is related to functionality in a prerelease version of dbt (delete if not applicable):
- [ ] I've added versioning components, as described in ["Versioning Docs"](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
- [ ] I've added a note to the prerelease version's [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`
- [ ] [Run link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

**Review by schottj — APPROVED** [review]
LGTM

File: website/docs/docs/dbt-cloud/release-notes/05-Sept-2002/liststeps-endpoint-deprecation.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: "List Steps API endpoint deprecation warning"
+id: "liststeps-endpoint-deprecation.md"
+description: "List Steps API deprecation"
+sidebar_label: "Deprecation: List Steps API endpoint"
+tags: [Sept-15-2022]
+---
+
+On October 14th, 2022 we are deprecating the [List Steps](https://docs.getdbt.com/dbt-cloud/api-v2#tag/Runs/operation/listSteps) API endpoint. From October 14th, any GET requests to this endpoint will fail. Please prepare to move off of the List Steps endpoint as soon as possible.
```
Comment (nghi-ly) [inline website/docs/docs/dbt-cloud/release-notes/05-Sept-2002/liststeps-endpoint-deprecation.md]:
```suggestion
On October 14th, 2022 dbt Labs is deprecating the [List Steps](https://docs.getdbt.com/dbt-cloud/api-v2#tag/Runs/operation/listSteps) API endpoint. From October 14th, any GET requests to this endpoint will fail. Please prepare to stop using the List Steps endpoint as soon as possible. 
```

File: website/docs/docs/dbt-cloud/release-notes/05-Sept-2002/liststeps-endpoint-deprecation.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+title: "List Steps API endpoint deprecation warning"
+id: "liststeps-endpoint-deprecation.md"
+description: "List Steps API deprecation"
+sidebar_label: "Deprecation: List Steps API endpoint"
+tags: [Sept-15-2022]
+---
+
+On October 14th, 2022 we are deprecating the [List Steps](https://docs.getdbt.com/dbt-cloud/api-v2#tag/Runs/operation/listSteps) API endpoint. From October 14th, any GET requests to this endpoint will fail. Please prepare to move off of the List Steps endpoint as soon as possible. 
+
+We will continue to maintain the [Get Run](https://docs.getdbt.com/dbt-cloud/api-v2#tag/Runs/operation/getRunById) endpoint, which is a viable alternative depending on the use case.
```
Comment (nghi-ly) [inline website/docs/docs/dbt-cloud/release-notes/05-Sept-2002/liststeps-endpoint-deprecation.md]:
```suggestion
dbt Labs will continue to maintain the [Get Run](https://docs.getdbt.com/dbt-cloud/api-v2#tag/Runs/operation/getRunById) endpoint, which is a viable alternative depending on the use case. 
```

File: website/docs/docs/dbt-cloud/release-notes/05-Sept-2002/liststeps-endpoint-deprecation.md (hunk 1 above)
Comment (nghi-ly) [reply website/docs/docs/dbt-cloud/release-notes/05-Sept-2002/liststeps-endpoint-deprecation.md]: sugg:
- "we" can be ambiguous. is it "dbt labs" that is being referred or is it "dbt labs + the reader" that's being referred? this ambiguity can make it harder to translate. don't avoid using it (as it makes sense to use it in certain situations) but use it sparingly. 
- "move off" might be a USAn colloquialism, not totally sure, but i do wonder how that'll translate to international users

**Review by nghi-ly — APPROVED** [review]
looks great!

File: website/docs/docs/dbt-cloud/release-notes/05-Sept-2002/liststeps-endpoint-deprecation.md (hunk 1 above)
Comment (mirnawong1) [reply website/docs/docs/dbt-cloud/release-notes/05-Sept-2002/liststeps-endpoint-deprecation.md]: ah yes great flag Ly, thank you for that and have committed your suggestions!

**Comment by mirnawong1** [comment]
thank you all, will merge!

**Pull request merged**